### PR TITLE
Updated HM Level / Added dkp and item pools for every raid / Added missing archer icon

### PR DESCRIPTION
**Pull request by Akizo96**

### CODE REVIEW

File: language/english.php [hunk 1]
```diff
@@ -28,6 +28,10 @@
         'cerulean_order' => 'The Cerulean Order',
         'crimson_legion' => 'The Crimson Legion',
     ),
+    'regions' => array(
+        'eu' => 'Europe',
+        'na' => 'North America'
```
Comment (Bloodsoul) [inline language/english.php]:
```suggestion
        'na' => 'North America',
```

File: language/german.php [hunk 2]
```diff
@@ -28,6 +28,10 @@
         'cerulean_order' => 'Der Coelinorden',
         'crimson_legion' => 'Die Karminlegion',
     ),
+    'regions' => array(
+        'eu' => 'Europa',
+        'na' => 'Nord Amerika'
```
Comment (Bloodsoul) [inline language/german.php]:
```suggestion
        'na' => 'Nordamerika',
```

File: language/german.php [hunk 3]
```diff
@@ -64,6 +68,14 @@
         'uc_class' => 'Klasse',
         'uc_level' => 'Level',
         'uc_hongmoon' => 'Hongmoon Level',
+        'uc_region' => 'Region',
+
+        'raid_bt' => 'Himmelsspitze',
+        'raid_vt' => 'Tempel der Winde',
+        'raid_sk' => 'Festung des Ursurpators',
```
Comment (Bloodsoul) [inline language/german.php]:
```suggestion
        'raid_sk' => 'Festung des Usurpators',
```

File: bs.class.php [hunk 4]
```diff
@@ -76,10 +84,10 @@ class bs extends game_generic {
                 'parent' => array(
                     'race' => array(
                         0 => 'all',                // Unknown
-                        1 => array(0,2,4,5,9,11),  // Gon
-                        2 => array(0,3,4,7,8,10),  // Lyn
-                        3 => array(0,1,4,5,9,10,12),  // Yun
-                        4 => array(0,1,5,6,8,9,10,11,12),  // Jin
+                        1 => array(0, 2, 4, 5, 9, 11),  // Gon
+                        2 => array(0, 3, 4, 7, 8, 10),  // Lyn
```
Comment (Bloodsoul) [inline bs.class.php]:
```suggestion
                        2 => array(0, 3, 4, 6, 7, 8, 10),  // Lyn
```

**Review by Bloodsoul — CHANGES_REQUESTED** [review]
Thank you very much for your pull-request. I would suggest only a few small changes prior to merging.

File: bs.class.php (hunk 4 above)
Comment (Bloodsoul) [reply bs.class.php]: As of this patch notes from february, Lyn should be able to be Assassins now. Is this correct?
https://www.bladeandsoul.com/en/news/shroud-of-the-assassin-patch-notes/

**Review by Bloodsoul — APPROVED** [review]

**Comment by Akizo96** [comment]
Hey, i commited the requested changes and yea, Assassin is now also available for Lyn

**Review by Bloodsoul — APPROVED** [review]
Thank you very much.

**Pull request merged**